code/game/jobs/job/command.dm

```diff
@@ -1,5 +1,5 @@
 /datum/job/captain
-	title = "Site Director"
+	title = "Station Commander"
```

Comment:
Job title changes like this need to be handled differently, you're changing the name on every map here.

Reply:
Oops, that never crossed my mind for some reason. Will resolve.

Reply:
Should be fixed now!